udata/frontend/markdown.py

```diff
@@ -40,7 +40,7 @@ def __call__(self, stream):
             styles=current_app.config['MD_ALLOWED_STYLES'],
             strip_comments=False)
         # Turn markdown to HTML.
-        ast = self.parser.parse(stream)
+        ast = self.parser().parse(stream)
```

Comment:
Should we not keep this as a property?
If not maybe changing the line below from `renderer` to `renderer()` would make it uniform.